sunpy/util/tests/test_metadata.py

```diff
@@ -0,0 +1,205 @@
+import pytest
+
+from sunpy.util.metadata import MetaDict
+
+
+class Base:
+
+    def setup_method(self):
+
+        self.seas = [['labrador', 'americas'],
```

Comment:
Generally when using pytest instead of writing classes, you write functions and use pytest fixtures to provide input to the tests. http://doc.pytest.org/en/latest/fixture.html